sampleapp/build.gradle

```diff
@@ -0,0 +1,50 @@
+apply plugin: 'com.android.application'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-android-extensions'
+
+android {
+    compileSdkVersion rootProject.ext.targetSdkVersion
+
+    defaultConfig {
+        applicationId "com.zendesk.sampleapp"
```

Comment:
I'd shove a `.connect.` or `.outbound.` or whatever in there too. Something like `com.zendesk.connect.sample`.

Reply:
Fair point, it does seem a little too general